regression/goto-analyzer/intervals_01/main.c

```diff
@@ -1,3 +1,4 @@
+#include <assert.h>
```

Comment:
Ok, but how does this relate to the PR? Adding a body to the commit message might be really helpful to explain this.